src/h3/api/basic_int/__init__.py

```diff
@@ -467,7 +467,7 @@ def uncompact_cells(cells, res):
     return _out_collection(hu)
 
 
-def h3shape_to_cells(h3shape, res):
+def h3shape_to_cells(h3shape, res, flags=0, experimental=False):
```

Comment:
I'm not sure if this should be the final form of the API. For example, we might not use integers for the flags, and we may drop the `experimental` option.

Could we leave `h3shape_to_cells` unchanged, and just create a `h3shape_to_cells_experimental` where we're free to change the API?